File: openapi-generator/dart/lib/model/asset_reefer_response_reefer_stats_return_air_temp.dart
part of openapi.api;

class AssetReeferResponseReeferStatsReturnAirTemp {
  /* Return air temperature in millidegree Celsius. */
  int tempInMilliC = null;
  /* Timestamp in Unix milliseconds since epoch. */
  int changedAtMs = null;
  AssetReeferResponseReeferStatsReturnAirTemp();

  @override
  String toString() {
    return 'AssetReeferResponseReeferStatsReturnAirTemp[tempInMilliC=$tempInMilliC, changedAtMs=$changedAtMs, ]';
  }

  AssetReeferResponseReeferStatsReturnAirTemp.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    if (json['tempInMilliC'] == null) {
      tempInMilliC = null;
    } else {
          tempInMilliC = json['tempInMilliC'];
    }
    if (json['changedAtMs'] == null) {
      changedAtMs = null;
    } else {
          changedAtMs = json['changedAtMs'];
    }
  }

  Map<String, dynamic> toJson() {
    return {
      'tempInMilliC': tempInMilliC,
      'changedAtMs': changedAtMs
    };
  }

  static List<AssetReeferResponseReeferStatsReturnAirTemp> listFromJson(List<dynamic> json) {
    return json == null ? new List<AssetReeferResponseReeferStatsReturnAirTemp>() : json.map((value) => new AssetReeferResponseReeferStatsReturnAirTemp.fromJson(value)).toList();
  }

  static Map<String, AssetReeferResponseReeferStatsReturnAirTemp> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, AssetReeferResponseReeferStatsReturnAirTemp>();
    if (json != null && json.length > 0) {
      json.forEach((String key, dynamic value) => map[key] = new AssetReeferResponseReeferStatsReturnAirTemp.fromJson(value));
    }
    return map;
  }
}

File: swagger-codegen/csharp-dotnet2/src/main/CsharpDotNet2/samsara/Model/AssetReeferResponseReeferStatsReturnAirTemp.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace samsara.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class AssetReeferResponseReeferStatsReturnAirTemp {
    /// <summary>
    /// Timestamp in Unix milliseconds since epoch.
    /// </summary>
    /// <value>Timestamp in Unix milliseconds since epoch.</value>
    [DataMember(Name="changedAtMs", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "changedAtMs")]
    public long? ChangedAtMs { get; set; }

    /// <summary>
    /// Return air temperature in millidegree Celsius.
    /// </summary>
    /// <value>Return air temperature in millidegree Celsius.</value>
    [DataMember(Name="tempInMilliC", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "tempInMilliC")]
    public long? TempInMilliC { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class AssetReeferResponseReeferStatsReturnAirTemp {\n");
      sb.Append("  ChangedAtMs: ").Append(ChangedAtMs).Append("\n");
      sb.Append("  TempInMilliC: ").Append(TempInMilliC).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
